src/components/Posts/ComposeReply.tsx

```diff
@@ -74,6 +76,24 @@ const ComposeReply = ({
       console.log(err);
     }
   };
+  const [showEmojiPicker, setShowEmojiPicker] = useState(false);
+  const toggleEmojiPicker = () => {
```

Comment:
inline setState works better I think..but tbh this is a design choice. If we want to say hey let's not do any inline, we should be consistent with the rest of our components.